table.rst

```diff
@@ -0,0 +1,43 @@
+
+Table for astype
+
+
++-------------------+-------------------------+
+|  S1               |  Dtype                  |
++===================+=========================+
+|  int              |  Int8Dtype,Int16Dtype,  |
+|                   |  Int32Dtype,Int54Dtype, |
++-------------------+-------------------------+
+|  str              | StringDtype,str         |
++-------------------+-------------------------+
+|  bytes            | BolleanDtype            |
++-------------------+-------------------------+
+|  float            |Float32Dtype,Float64Dtype|
++-------------------+-------------------------+
+|  complex          |                         |
++-------------------+-------------------------+
+|  Timestamp        |  Timestamp              |
++-------------------+-------------------------+
+|  Timedelta        | Timedelta,TimedeltaIndex|
++-------------------+-------------------------+
+|   Period          | Period, PeriodDtype     |
+|                   | PeriodIndex             |
++-------------------+-------------------------+
+| np.timedelta64    |                         |
++-------------------+-------------------------+
+|                   | np.datetime64           |
```

Comment:
should have `np.datetime64` in the left column, `np.datetime64` and `pd.DatetimeTZDtype()` in the right column